ganttproject/src/main/java/biz/ganttproject/storage/cloud/GPCloudHttpImpl.kt

```diff
@@ -262,18 +267,27 @@ class WebSocketClient {
   private val structureChangeListeners = mutableListOf<(Any) -> Unit>()
   private val lockStatusChangeListeners = mutableListOf<(ObjectNode) -> Unit>()
   private val contentChangeListeners = mutableListOf<(ObjectNode) -> Unit>()
+  private val xlogReceivedListeners = mutableListOf<(ObjectNode) -> Unit>()
   private var listeningDocument: GPCloudDocument? = null
 
-  fun start() {
+  private fun getWebSocketUrl(): String {
+    return if (isColloboqueLocalTest()) {
+      "ws://localhost:${System.getProperty("colloboquePort", "9001")}"
+    } else {
+      GPCLOUD_WEBSOCKET_URL
+    }
+  }
 
+  fun start() {
     LOG.debug("WebSocket started")
-    val req = Request.Builder().url(GPCLOUD_WEBSOCKET_URL).build()
+    val req = Request.Builder().url(getWebSocketUrl()).build()
     this.websocket?.close(1000, "Reset Websocket")
     this.heartbeatFuture?.cancel(true)
-    wsListener = WebSocketListenerImpl(GPCloudOptions.websocketAuthToken, this::onAuthDone, this::onMessage, this::onClose)
+    val wsListener = WebSocketListenerImpl(GPCloudOptions.websocketAuthToken, this::onAuthDone, this::onMessage, this::onClose)
+    this.wsListener = wsListener
     this.websocket = OkHttpClient.Builder()
-      .connectionSpecs(listOf(ConnectionSpec.COMPATIBLE_TLS))
-      .build().newWebSocket(req, this.wsListener)
+      .connectionSpecs(listOf(ConnectionSpec.COMPATIBLE_TLS, ConnectionSpec.CLEARTEXT))
```

Comment:
maybe wrap this into isLocal test as well? Because this probably means that should TLS in prod fail, we'll start sending data anyway in clear text.